docs/docs/reference/changed-features/implicit-conversions.md

```diff
@@ -26,6 +26,8 @@ Defining an implicit conversion will emit a warning unless the import
 `scala.language.implicitConversions` is in scope, or the flag
 `-language:implicitConversions` is given to the compiler.
 
+Conversion can no longer be defined as a value of type `T => S`.
```

Comment:
It is no longer sufficient to define an implicit value of type `T => S`. The conversion must be supplied as an `implicit def`, as shown, or a value of type `Conversion[T, S]`, which is a subtype of `T => S`.

Reply:
That's why it `can no longer`

Reply:
The diff might be confusing because `Conversion` is `T => S`. As an aside, maybe there is a reason to use `T` and `S`, but it feels like the conversion is going backward. Unfortunately `T => U` has the connotation of `Unit`. Too bad it's not just `A => B` which is free of associations. Not to say free-associating.

Reply:
I see my first comment was intended as alternative text. Implicitly, I meant that "can no longer" is not explanatory enough. (I'm not sure if the downvote was to object that it is explanatory enough, or if it was about my comment about choice of letters.) Maybe there is a place in a FAQ for the clarification.